tests/e2e/br/br.go

```diff
@@ -216,6 +225,81 @@ var _ = ginkgo.Describe("Backup and Restore", func() {
 			})
 		}
 	})
+
+	ginkgo.It("backup and restore with mixed bucket and prefix", func() {
+		middlePath := "mid"
+		expectedPrefix := ""
+
+		ns := f.Namespace.Name
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		tcase := newTestCase(utilimage.TiDBLatest, utilimage.TiDBLatest, typeBR)
+		tcase.configureBackup = func(backup *v1alpha1.Backup) {
+			backup.Spec.StorageProvider.S3.Bucket = path.Join(backup.Spec.StorageProvider.S3.Bucket, middlePath) // bucket add suffix
+			expectedPrefix = path.Join(middlePath, backup.Spec.StorageProvider.S3.Prefix)
```

Comment:
Is it better to move this to the postBackup block?

Reply:
the function postBackup exec after creating Backup, so we should configure Backup in the function configureBackup

Reply:
I mean the variable `expectedPrefix `.

Reply:
oh, yes 😄